tests/h/search/test_logging_filters.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""Test logging filters."""
+from __future__ import unicode_literals
+
+import pytest
+import logging
+from requests.exceptions import ReadTimeout
+from h.search.logging_filters import FilterExceptions
+
+
+class TestLoggingFilters(object):
+    """Test logging filters."""
+
+    def test_filter_bad_level(self):  #pylint: disable=no-self-use
```

Comment:
I'm afraid our linting story is not super-clear at the moment for new devs. [flake8](http://flake8.pycqa.org/en/latest/) is the one tool that all of us run locally, is quick and I've proposed we make part of the build process ([feedback on this welcome 🙂](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1517481793000097))

Reply:
Ah ok. This was coming from prospector I believe. I can remove it.